ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-commands.sh

```diff
@@ -92,11 +92,29 @@ fi
 cat >> "${CONFIG}" << EOF
 compute:
 - name: worker
+  replicas: 3
   platform:
     openstack:
       type: ${OPENSTACK_COMPUTE_FLAVOR}
       zones: ${ZONES_STR}
-  replicas: 3
+EOF
+if [[ ${ADDITIONAL_WORKERS_NETWORKS} != "" ]]; then
+    cat >> "${CONFIG}" << EOF
+      additionalNetworkIDs:
+EOF
+    for network in $(echo "${ADDITIONAL_WORKERS_NETWORKS}" | tr " " "\n"); do
```

Comment:
Isn't it equivalent to the following?
```
for network in $ADDITIONAL_WORKERS_NETWORKS; do
```

Reply:
ok i'll change it, maybe in a second batch if that's fine